api/src/main/java/org/apache/iceberg/view/SQLViewRepresentation.java

```diff
@@ -41,8 +40,8 @@ default Type type() {
   /** The default namespace when the view is created. */
   Namespace defaultNamespace();
 
-  /** The query output schema at version create time, without aliases. */
-  Schema schema();
```

Comment:
why change it to schema ID? If we know the schema ID, we should also be able to retrieve the Schema object